frontend/website/src/components/chatroom.svelte

```diff
@@ -100,10 +159,17 @@
 		<span class="chat-title">Chat</span>
 	</div>
 	<div class="messages" bind:this={messagesEl}>
-		{#each messages as message (message.svelte_id)}
-			<div class="message" class:odd={message.svelte_id % 2}>
-				<span class="message-sender" style={`color: ${message.color}`}>{message.sender}:</span>
-				<span class="message-text">{message.text}</span>
+		{#each messages as message}
+			<div class="message">
+				{#if message.messageType === "message"}
+					<span class="message-sender" style={`color: ${message.metadata.color}`}
```

Comment:
Where does the color come from?
At the moment user's dont have a color in the database.

Reply:
Its right now hardcoded on backend to teal

Reply:
ok thats fine